app/services/activitypub/synchronize_followers_service.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class ActivityPub::SynchronizeFollowersService < BaseService
+  include JsonLdHelper
+  include Payloadable
+
+  def call(account, partial_collection_url)
+    @account = account
+
+    items = collection_items(partial_collection_url)
+    return if items.nil?
+
+    @expected_followers = items.map { |uri| ActivityPub::TagManager.instance.uri_to_resource(uri, Account) }.compact
+    # TODO: what to do with unresolvable accounts?
```

Comment:
It should not be possible for "our" accounts to be unfindable since the `Account` record is always kept

Reply:
Yeah. Well, we do in some cases allow removing `Account`, I think? But in any case, the case should be exceedingly rare, and if it occurs there's nothing much we can do since we don't have an Account to sign an `Undo` with. Updated the comment.